Sources/BLTNItemManager.swift

```diff
@@ -97,7 +97,12 @@ import UIKit
      */
 
     @objc public var allowsSwipeInteraction: Bool = true
+    
+    /**
+     * Tells us if a bulletin is currently being shown. Defaults to false
+     */
 
+    @objc public var isShowingBulletin: Bool { return isPrepared }
```

Comment:
I think it would be more accurate to check if the bulletin view controller is currently presented.

```suggestion
    @objc public var isShowingBulletin: Bool { 
        return bulletinController?.presentingViewController != nil
    }
```